eta/core/serial.py

```diff
@@ -670,32 +705,325 @@ def from_dict(cls, d):
 
             # Parse reflectively
             cls = etau.get_class(d["_CLS"])
-            ele_cls = etau.get_class(d[cls._ELE_CLS_FIELD])
+
         else:
             # Validates the cls settings
             cls()
-            # Parse using provided class
-            ele_cls = cls._ELE_CLS
 
-        return cls(**{
-            cls._ELE_ATTR: [ele_cls.from_dict(dd) for dd in d[cls._ELE_ATTR]]
-        })
+
+class DirectoryContainer(Container):
+    '''Container that represents a list of serializable objects in a
+    directory on disk. Any accessing of an element in the list causes a READ
+    from disk and updating any elements in the list REQUIRES setting the item
+    explicitly (and causes a WRITE to disk).
```

Comment:
I just remembered one more feature request: implement `to_container` and `from_container` that will convert between `Big<ContainerClass>` and `<ContainerClass>`. The convention would be that these classes would be defined in the same module. If `<ContainerClass>` does not exist, the user gets an error message.

`to_container()` would be the stylistic thing for users of these classes, not `in_memory_list = [e for e in c]`